src/spork.h

```diff
@@ -75,15 +88,41 @@ class CSporkMessage
         }
     }
 
+    /**
+     * GetHash returns the double-sha256 hash of the serialized spork message. The
+     * signature field is not included in the serialization.
+     */
     uint256 GetHash() const;
+
+    /**
+     * GetSignatureHash is currently an alias for GetHash, and I'm not sure why it
```

Comment:
>I'm not sure why

`The signature field is not included in the serialization.` - that's why in this case ;)